crates/feldera-types/src/format/avro.rs

```diff
@@ -54,3 +60,60 @@ pub struct AvroEncoderConfig {
     /// password-based authentication (see `registry_username` and `registry_password`).
     pub registry_authorization_token: Option<String>,
 }
+
+/// Avro output format configuration.
+#[derive(Clone, Serialize, Deserialize, Debug, Default, ToSchema)]
+#[serde(deny_unknown_fields)]
+pub struct AvroParserConfig {
```

Comment:
If it's output why is it called Parser?
Because it's the parser that receives this data?

Reply:
This one is actually input.  The typo is below, where I describe the schema as output schema (copy/paste error).